allennlp/data/data_loaders/multi_process_data_loader.py

```diff
@@ -75,7 +75,7 @@ class MultiProcessDataLoader(DataLoader):
         to `__iter__()`.
 
     num_workers: `int`, optional (default = `0`)
-        The number of workers to use to read `Instance`s in parallel.
+        The number of workers to use to read `Instances` in parallel.
```

Comment:
That was correct before, no? `Instance` is the type, so `Instance`s is many of them?

Reply:
![PpV0evc](https://user-images.githubusercontent.com/920638/104362114-3be52f00-54c8-11eb-94d0-58385850c6cd.png)